packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -1766,9 +1771,9 @@ def _get_non_empty_values(self) -> List:
         non_empty_values = []
 
         for payload in self.collection.values():
-            attribute_value = getattr(payload, self.payload_attribute, None)
-            if attribute_value is not None:
-                non_empty_values.append(attribute_value)
+            # attribute_value = getattr(payload, self.payload_attribute, None)
+            if payload.data:
+                non_empty_values.append(payload.data)
```

Comment:
will remove outcommented code later

Reply:
This code is no longer equivalent to before. IT will be hard to do so many changes in one go